source/_integrations/ezviz.markdown

```diff
@@ -17,231 +19,11 @@ It also exposes an RTSP stream, by using the local camera IPs (so the device hos
-      top: 25px
-      right: 0px
-    tap_action:
-      action: call-service
-      service: camera.ezviz_switch_follow_move_off
-      service_data:
-        entity_id: camera.c6n_d12345678
-      confirmation: true
-    type: icon
-  - icon: 'mdi:eye'
-    style:
-      background: "rgba(255, 255, 255, 0.5)"
-      right: 25px
-      top: 50px
-    tap_action:
-      action: call-service
-      service: camera.ezviz_switch_privacy_off
-      service_data:
-        entity_id: camera.c6n_d12345678
-      confirmation: true
-    type: icon
-  - icon: 'mdi:eye-off'
-    style:
-      background: "rgba(255, 255, 255, 0.5)"
-      right: 0px
-      top: 50px
-    tap_action:
-      action: call-service
-      service: camera.ezviz_switch_privacy_on
-      service_data:
-        entity_id: camera.c6n_d12345678
-      confirmation: true
-    type: icon
-  - icon: 'mdi:volume-high'
-    style:
-      background: "rgba(255, 255, 255, 0.5)"
-      right: 25px
-      top: 75px
-    tap_action:
-      action: call-service
-      service: camera.ezviz_switch_audio_off
-      service_data:
-        entity_id: camera.c6n_d12345678
-      confirmation: true
-    type: icon
-  - icon: 'mdi:volume-off'
-    style:
-      background: "rgba(255, 255, 255, 0.5)"
-      right: 0px
-      top: 75px
-    tap_action:
-      action: call-service
-      service: camera.ezviz_switch_audio_on
-      service_data:
-        entity_id: camera.c6n_d12345678
-      confirmation: true
-    type: icon
-  - icon: 'mdi:led-on'
-    style:
-      background: "rgba(255, 255, 255, 0.5)"
-      right: 25px
-      top: 100px
-    tap_action:
-      action: call-service
-      service: camera.ezviz_switch_state_off
-      service_data:
-        entity_id: camera.c6n_d12345678
-      confirmation: true
-    type: icon
-  - icon: 'mdi:led-off'
-    style:
-      background: "rgba(255, 255, 255, 0.5)"
-      right: 0px
-      top: 100px
-    tap_action:
-      action: call-service
-      service: camera.ezviz_switch_state_on
-      service_data:
-        entity_id: camera.c6n_d12345678
-      confirmation: true
-    type: icon
-  - icon: 'mdi:brightness-4'
-    style:
-      background: "rgba(255, 255, 255, 0.5)"
-      right: 25px
-      top: 100px
-    tap_action:
-      action: call-service
-      service: camera.ezviz_switch_ir_off
-      service_data:
-        entity_id: camera.c6n_d12345678
-      confirmation: true
-    type: icon
-  - icon: 'mdi:brightness-5'
-    style:
-      background: "rgba(255, 255, 255, 0.5)"
-      right: 0px
-      top: 100px
-    tap_action:
-      action: call-service
-      service: camera.ezviz_switch_ir_on
-      service_data:
-        entity_id: camera.c6n_d12345678
-      confirmation: true
-    type: icon
-entity: camera.c6n_d12345678
-type: picture-elements
-```
+You can also change the camera options should you need to access a high or low res stream. Generally though it will just work without any modification to options.
```

Comment:
```suggestion
You can also change the camera options should you need to access a high or low res stream. Generally, though it will just work without any modification to options.
```

Which options? Where do I find them?